track_info.c

```diff
@@ -46,8 +46,12 @@ static void track_info_free(struct track_info *ti)
 
 struct track_info *track_info_new(const char *filename)
 {
+	static uint64_t cur_uid = 0;
+	cur_uid++;
```

Comment:
Sorry, I don't get it: what race is possible when using the pointer value, but not possible with `static int` approach?